src/cmd/flux-start.c

```diff
@@ -482,6 +503,21 @@ int execvp_argz (char *argz, size_t argz_len)
     return -1;
 }
 
+bool system_instance_is_running (void)
```

Comment:
This is fine, but i couldn't help but wonder if there's a better way or "correct way" to do this, like perhaps calling systemd directly?

Reply:
Heh, maybe, although I don't have the first idea of how to do that, and I would imagine it would be more complex...

Reply:
FWIW, this seems fine to me since it would work even if the system instance is started without systemd or via some other process management scheme (e.g. in cloud scenarios if that is ever a thing?)

What struck me here is that this duplicates code that is already present in `flux_open(3)` -- would it be simpler to just unset `FLUX_URI` and then call `flux_open (NULL, 0)`?

Reply:
Good point!  Will fix.